### ISSUE TITLE
7800xt are these numbers realistic or is that a bug (idle Clocks sub-100MHz)?

### ISSUE THREAD

**Issue by My1** [marazmista/radeon-profile#301]
allegedly it goes down to sub-100MHz when idling even on a 4K screen, the deepest point i saw was 63 MHz core and 96 Memory clock.

<img width="339" alt="image" src="https://github.com/user-attachments/assets/8ddd5913-67c3-49c6-898b-d56574502f72">

are these cards really able to decently supply idle and jump back up that strongly or is something broken here?

**Comment by yjfguhd** [comment]
Yep. I'm now averaging around 20 MHz core and a steady 96 MHz memory frequencies, UHD display at 60Hz. Measured outside of radeon-profile. It jumps higher when I play games.
